src/components/Footer/data.ts

```diff
@@ -6,7 +6,7 @@ import FusionIcon from 'public/assets/svg/makerdao_fusion.svg';
 import SwitchboardIcon from 'public/assets/svg/makerdao_switchboard.svg';
 import PowerhouseIcon from 'public/assets/svg/powerhouse.svg';
 import RedditIcon from 'public/assets/svg/reddit.svg';
-import TwitterIcon from 'public/assets/svg/twitter.svg';
+import XIcon from 'public/assets/svg/xIcon.svg';
```

Comment:
This icon was already created so we should utilize the one we already have instead of creating it again

Reply:
you are right, let me check it